src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/DepartmentRequestsTests.cs

```diff
@@ -569,24 +560,15 @@ public async Task GetTimeline_ShouldNotIncludeTaskDetailsForOtherResourceOwner()
                 absence = absenceResp.Value;
             }
 
-            var resourceOwner = fixture.AddProfile(FusionAccountType.Employee);
-            resourceOwner.IsResourceOwner = true;
-            resourceOwner.FullDepartment = "TPD TST QWE";
-            resourceOwner.Department = "TPD TST QWE";
-
-            LineOrgServiceMock.AddTestUser()
-                .MergeWithProfile(resourceOwner)
-                .AsResourceOwner()
-                .WithDepartment("TPD TST QWE")
-                .SaveProfile();
+            var resourceOwner = fixture.AddResourceOwner("TPD TST QWE");
 
             using (var userScope = fixture.UserScope(resourceOwner))
             {
                 var timelineStart = new DateTime(2020, 03, 01);
                 var timelineEnd = new DateTime(2020, 03, 31);
 
                 var response = await Client.TestClientGetAsync<TestResponse>(
-                    $"/departments/{user.Department}/resources/personnel/?$expand=timeline&{ApiVersion}&timelineStart={timelineStart:O}&timelineEnd={timelineEnd:O}"
+                    $"/departments/{TimelineDepartment}/resources/personnel/?$expand=timeline&{ApiVersion}&timelineStart={timelineStart:O}&timelineEnd={timelineEnd:O}"
```

Comment:
Some tests had to be changed as `user.FullDepartment` no longer points to the same org unit, but one level up....